lib/preprocessor.js

```diff
@@ -80,35 +79,36 @@ function createCoveragePreprocessor (logger, helper, basePath, reporters, covera
 
   var sourceMapStore = globalSourceMapStore.get(basePath)
 
-  var instrumentersOptions = _.reduce(instrumenters, function getInstrumenterOptions (memo, instrument, name) {
-    memo[name] = {}
+  const instrumentersOptions = Object.keys(instrumenters).reduce((memo, key) => {
+    memo[key] = {}
 
     if (coverageReporter && coverageReporter.instrumenterOptions) {
-      memo[name] = coverageReporter.instrumenterOptions[name]
+      memo[key] = coverageReporter.instrumenterOptions[key]
     }
 
     return memo
   }, {})
 
   // if coverage reporter is not used, do not preprocess the files
-  if (!_.includes(reporters, 'coverage')) {
+  if (!reporters.includes('coverage')) {
     return function (content, _, done) {
       done(content)
     }
   }
 
   // check instrumenter override requests
   function checkInstrumenters () {
-    return _.reduce(instrumenterOverrides, function (acc, literal, pattern) {
-      if (!_.includes(_.keys(instrumenters), String(literal))) {
+    const keys = Object.keys(instrumenters)
+    return Object.values(instrumenterOverrides).some(literal => {
+      const notIncluded = !keys.includes(String(literal))
+      if (notIncluded) {
         log.error('Unknown instrumenter: %s', literal)
-        return false
       }
-      return acc
-    }, true)
+      return notIncluded
+    })
   }
 
-  if (!checkInstrumenters()) {
+  if (checkInstrumenters()) {
```

Comment:
How can dropping `!` be ok here?

Reply:
checkInstrumenters also changed that mean checkInstrumenters return true if we don't find instrument